### PR TITLE
Fall24 Review

### PR DESCRIPTION
**Pull request by FabrizioMoggio**
- Commonalities version added
- Base URL removed
- version moved to wip
- Release notes removed
- Contact info removed

#### What type of PR is this?
* documentation
#### What this PR does / why we need it:
To be compliant with Commonalities

#### Which issue(s) this PR fixes:
 https://github.com/camaraproject/CallForwardingSignal/issues/66
 https://github.com/camaraproject/CallForwardingSignal/issues/69
https://github.com/camaraproject/CallForwardingSignal/issues/71
https://github.com/camaraproject/CallForwardingSignal/issues/70
https://github.com/camaraproject/CallForwardingSignal/issues/76

### CODE REVIEW

File: code/API_definitions/Call_Forwarding_Signal.yaml [hunk 1]
```diff
@@ -147,9 +141,14 @@ info:
     (FAQs will be added in a later version of the documentation)
     # Release Notes
     First draft for the CFS API.
```
Comment (hdamker) [inline code/API_definitions/Call_Forwarding_Signal.yaml:143]: I don't think that release notes belong here. They would be difficult to maintain.

Comment (FabrizioMoggio) [reply code/API_definitions/Call_Forwarding_Signal.yaml:143]: I agree. 

They where requested here: https://github.com/camaraproject/Commonalities/blob/main/documentation/API-DocumentationTemplate.md

Currently we have changelog and release management, I think we can remove them from the YAML

**Comment by hdamker** [comment]
@FabrizioMoggio: there is a general challenge with this PR - you have created already a r1.1 tag with the API Version 0.2.0-rc.1 which you can't and shouldn't change anymore (the tagged commit is https://github.com/camaraproject/CallForwardingSignal/commit/33f21b2b6fa73276cb33daa59d864b6facb70bfa).

> Once a versioned package has been released, the contents of that version MUST NOT be modified. Any modifications MUST be released as a new version. (https://semver.org/#spec-item-3)

The way forward is to consider a second release candidate with API Version 0.2.0-rc.2 and r1.2 as tag (that will come with a new section within the changelog, update of README.md, checklist, ...). That should be created in a separate Release PR.

But I wouldn't already now create the second release candidate but wait for further feedback on the first release candidate and some more substantial fixes then just the formal ones. You can for example open a "scope" issue for the rc.2 (like Commonalities did in https://github.com/camaraproject/Commonalities/issues/253

This current PR here which is fixing #66 and #69 should reset the version in the meantime to "wip" as there are changes to the YAML included which shouldn't be part of a release PR. But you can keep it also open for a while to collect more feedback on rc.1.

**Review by chinaunicomyangfan — APPROVED** [review]
LGTM

**Review by hdamker — COMMENTED** [review]
Looks good from release management

**Comment by FabrizioMoggio** [comment]
> Looks good from release management

Can I move forward with the Release Tag?

**Comment by hdamker** [comment]
> Can I move forward with the Release Tag?

See #72 for next steps. Have only seen afterwards that that the changes for the release are spread across PRs.

**Pull request merged**